src/components/ProgressBar/component.tsx

```diff
@@ -0,0 +1,247 @@
+import * as React from 'react';
```

Comment:
Since this is mostly code from their github, please put in the comments at the top  `// original code here: https://github.com/sghall/react-compound-slider/blob/master/docs/src/demos/horizontal/components.tsx`